recommendations.md

```diff
@@ -60,3 +60,252 @@ On the catalog, this means that you should, where possible:
         runAsUser: 0 # root uid == 0
         privileged: true
   ```
+
+## Remember that there are other languages than sh and bash
+
+Yes, sh and bash are DSLs for running processes, but sometimes there
+are other languages more suited for what you're trying to do.  Tekton
+Pipelines' main positive attribute is the ability to have the right
+tool available for every step, and that includes the _interpreter_.
+Use python or another scripting languages when that is warranted.
+
+A python example:
+
+```
+  steps:
+    - name: foo
+      image: python:alpine
+      script: |
+        #!/bin/env python
```

Comment:
nit: maybe python3 :)

(/usr/bin/python is not found by default on fedora/ubuntu)